### PR TITLE
Support FeedBooks short stories genre

### PR DESCRIPTION
**Pull request by courte**
Supports the work done in NYPL-Simplified/content_server#116, including:
 - allowing URI-type identifiers to be differentiated by DataSource,
 - creating the FeedBooks DataSource at initialization,
 - passing bypass database sessions, and
 - trimming some long lines and unused variables.

### CODE REVIEW

File: model.py [hunk 1]
```diff
@@ -4020,7 +4021,7 @@ def calculate_quality(self, identifier_ids, default_quality=0):
             self, operation=WorkCoverageRecord.QUALITY_OPERATION
         )
 
-    def assign_genres(self, identifier_ids, cutoff=0.15):
```
Comment (courte) [inline model.py]: `cutoff` wasn't being used.

File: scripts.py [hunk 2]
```diff
@@ -232,6 +235,10 @@ def arg_parser(cls):
             help='Process identifiers of this type. If IDENTIFIER is not specified, all identifiers of this type will be processed. If IDENTIFIER is specified, this argument is required.'
         )
         parser.add_argument(
+            '--identifier-data-source',
```
Comment (courte) [inline scripts.py]: This supports cases like Identifier.URI.

File: scripts.py [hunk 3]
```diff
@@ -232,6 +235,10 @@ def arg_parser(cls):
...
+            help='Process identifiers with this LicensePool.data_source'
```
Comment (leonardr) [inline scripts.py]: This is a little confusing to me. I would say "Process only identifiers which have a license pool associated with this data source."

**Comment by leonardr** [comment]
:+1: with one minor clarification.

**Comment by leonardr** [comment]
:+1:

**Comment by courte** [comment]
Since we're going to keep working with FeedBooks, we may want to create a FeedBooksClassifier at some point. Their identifiers have subject information that can be more detailed or direct than their subject names.

Some examples:
 - `FBFIC027040 - Gothic` and `FBFIC024010N - Gothic` are both gothic fiction, but "Gothic" alone is too general a keyword to make the assumption.
 - `FBDRA011000N - Greek` is a Greek drama, while `FBJUV022020 - Greek & Roman` is a juvenile work with Greek / Roman aspects.
 - `FBFIC004030N - Modern (<1799)` and `FBFIC014040N - Modern (<1799)` are fiction. `FBPOE009000N - Modern (<1799)` is poetry, and `FBDRA050000N - Modern (<1799)` is drama.
 - So on and so forth.

**Pull request merged**